### PR TITLE
Minor doc updates and bootstrap cleanups

### PR DESCRIPTION
**Pull request by rspier**

### CODE REVIEW

File: bootstrap/selfconfig-root [hunk 1]
```diff
@@ -71,6 +71,7 @@ my @required_debs = qw(
  libssl-dev
  nginx
  python3-certbot-nginx
+ rename
```
Comment (rjbs) [inline bootstrap/selfconfig-root:75]: Okay, but out of curiosity where is it used?

**Review by rjbs — APPROVED** [review]
👍🏽

File: bootstrap/selfconfig-root (hunk 1 above)
Comment (rspier) [reply bootstrap/selfconfig-root:75]: It's used by journal-length-exporter, which is non-important monitoring, so isn't part of the normal bootstrap.  Maybe it shouldn't be here?

Comment (rspier) [reply bootstrap/selfconfig-root]: I dropped it for now. It's not actually needed by pause.  (There is still an open question of how we want to track monitoring.)

**Review by wolfsage — APPROVED** [review]

**Pull request merged**